internal/rpc/jsonrpc/methods.go

```diff
@@ -2220,10 +2220,8 @@ func (s *Server) getMultisigOutInfo(ctx context.Context, icmd interface{}) (inte
 	}
 
 	// Get the list of pubkeys required to sign.
-	_, pubkeyAddrs, _, err := txscript.ExtractPkScriptAddrs(
-		0, p2shOutput.RedeemScript,
-		w.ChainParams(), true) // Yes treasury
-	if err != nil {
+	_, pubkeyAddrs := stdscript.ExtractAddrs(0 /* TODO */, p2shOutput.RedeemScript, w.ChainParams())
+	if len(pubkeyAddrs) == 0 {
```

Comment:
And this shouldn't be added either since a response with no addresses is still expected to keep the same behavior.